go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift-kni/cnf-features-deploy
 
-go 1.19
+go 1.20
```

Comment:
do we want to bump the golang version (do we need to bump the toolchain as well?) in the same commit and in the same PR?

Reply:
good question maybe we can have the go bump in a separate PR

Reply:
@liornoy please go update from this PR
BTW it seems like it needs more work anyway (just a headsup for #1634)
```
Trying to pull registry.ci.openshift.org/openshift/release:golang-1.20...
Warning: Pull failed, retrying in 5s ...
Trying to pull registry.ci.openshift.org/openshift/release:golang-1.20...
Warning: Pull failed, retrying in 5s ...
error: build error: failed to pull image: After retrying 2 times, Pull image still failed due to error: initializing source docker://registry.ci.openshift.org/openshift/release:golang-1.20: reading manifest golang-1.20 in registry.ci.openshift.org/openshift/release: manifest unknown
```